plugins/woocommerce/tests/e2e-pw/tests/shopper/account-email-receiving.spec.js

```diff
@@ -240,10 +240,10 @@ test.describe(
 					.click();
 
 				await expect(
-					await page
-						.getByText( 'Password reset email has been sent' )
-						.count()
-				).toBeGreaterThan( 0 );
+					page
+						.getByRole( 'alert' )
+						.getByText( 'Password reset email has been sent.' )
+				).toBeVisible();
```

Comment:
This was failing on the WPCOM environment. Anyway, updating it to look for an alert rather than verifying the >0 number is a much more convenient assertion.